tox.ini

```diff
@@ -23,8 +22,8 @@ deps =
     -r{toxinidir}/requirements-testing.txt
 
 [testenv:py38-pytestlatest-linters]
-deps = black
-commands = black --check --verbose setup.py pytest_factoryboy tests
+deps = pre-commit
```

Comment:
I don't really like the build to fail if a hook like trailing-whitespace fails.
I think that failing only if black fails is enough.

Reply:
we should fail the build if anything in pre-commit fails

Reply:
why?

Reply:
because that's the purpose of pre-commit

Reply:
(comment not related to this PR, but to `pre-commit`) it will be very nice to add some file (`CONTRIBUTING.md`) or section in `README.md` about how to contribute to the project in steps, so then we could force contributors to use `pre-commit` and run it in the CI job

Reply:
@skarzi I've requested to add pre-commit.ci it will automatically apply fixes to PRs

Reply:
looks like pre-commit.ci was enabled. we could remove this check from tox now

Reply:
let's remove this whole section then